Plugins/Json/src/SurfaceJsonConverter.cpp

```diff
@@ -25,6 +27,15 @@
 #include "Acts/Surfaces/StrawSurface.hpp"
 #include "Acts/Surfaces/TrapezoidBounds.hpp"
 
+void Acts::to_json(nlohmann::json& j, const Acts::SurfaceAndMaterial& surface) {
+  to_json(j, surface.first);
+  to_json(j, surface.second.get());
+}
+
+void Acts::to_json(nlohmann::json& j, const Acts::SurfacePointer& surface) {
+  to_json(j, *surface);
```

Comment:
Oh, now I understand ... I think if the `SurfaceJsonConverter` from the framework is in, this gets easier. I added another convenience function to write out a `const Surface& ref`